<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authentification</class>
 <widget class="QDialog" name="Authentification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: rgb(255, 255, 255);
border-radius: 20px;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icon.qrc">:/ICON/pngtree-medical-logo-vector-png-image_6713322.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>170</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(36, 156, 200);</string>
   </property>
   <property name="text">
    <string>MEDOC</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="L_user_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom: 1px solid rgb(227, 237, 243);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="L_mot_de_passe">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>252</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>325</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mot de passe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnconnexion">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>251</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>202</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 156, 200);
color: rgb(0, 0, 0);
border: none;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>se connecter</string>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="la_erreur">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
